### ISSUE TITLE
sysctl::apply always updates 2 resources

### ISSUE THREAD

**Issue by chasebolt** [svanzoest-cookbooks/sysctl#60]
is there a way to prevent this from updating 2 resources, on every chef run, from the notifies that happen with the apply recipe? we expect chef runs to always have 0 resources updated, until an actual change has happened.

**Comment by svanzoest** [comment]
certainly something to investigate. It is not something that we have spend any time on.

**Comment by flaccid** [comment]
This a (fairly minor) annoyance for me too, it would be good to solve so subsequent chef runs are not falsely reporting changed resources. For example in a provider its as simple as `new_resource.updated_by_last_action(true) if a_boolean_used_to_track_if_any_resources_where_actually_changed`.

An example chef run where sysctl is always being run and set:

```
  * sysctl_param[net.ipv4.conf.all.forwarding] action apply

  * execute[sysctl[net.ipv4.conf.all.forwarding]] action run
    - execute sysctl -w "net.ipv4.conf.all.forwarding=1"
  * sysctl_param[net.ipv6.conf.all.forwarding] action apply

  * execute[sysctl[net.ipv6.conf.all.forwarding]] action run
    - execute sysctl -w "net.ipv6.conf.all.forwarding=1"

  * ruby_block[save-sysctl-params] action run
    - execute the ruby block save-sysctl-params
```

**Comment by svanzoest** [comment]
This will be resolved when we commit #61
